src/models/Notifications.js

```diff
@@ -0,0 +1,16 @@
+import { Schema, model } from "mongoose";
+
+const schema = new Schema(
+  {
+    notification_id: { type: String, required: true },
```

Comment:
no notification_id needed here, mogodb will create the internal _id

Reply:
changed as suggested